docs/web/api-doc.md

```diff
@@ -219,3 +219,24 @@ HTTP error code | Reason
 500             | Internal server error. It is strongly recommended to send us the message and the link to it, as it is very likely a bug in our system.
 501             | Only a special list of vehicles is supported
 400             | Something was wrong in your request
+
+## Isochrone
+
+In addition to routing, the URL path to obtain an isochrone is `/isochrone`.
```

Comment:
Hmmh, when I read this again I think the original routing description should exactly the opposite:

> The URL of the local instance is http://localhost:8989
> The end point to obtain the route is /route

?